safe-logging-refactorings/src/main/java/com/palantir/logsafe/refactorings/DeprecatedLoggableExceptionAssertArgs.java

```diff
@@ -26,12 +27,12 @@
 
     @BeforeTemplate
     @SuppressWarnings("deprecation")
-    LoggableExceptionAssert<T> hasArgs(LoggableExceptionAssert<T> loggableExceptionAssert, Arg<?>... args) {
+    LoggableExceptionAssert<T> hasArgs(LoggableExceptionAssert<T> loggableExceptionAssert, @Repeated Arg<?> args) {
         return loggableExceptionAssert.hasArgs(args);
     }
 
     @AfterTemplate
-    LoggableExceptionAssert<T> containsArgs(LoggableExceptionAssert<T> loggableExceptionAssert, Arg<?>... args) {
+    LoggableExceptionAssert<T> containsArgs(LoggableExceptionAssert<T> loggableExceptionAssert, @Repeated Arg<?> args) {
         return loggableExceptionAssert.containsArgs(args);
     }
```

Comment:
Would be nice if we had a framework to make these easy to quickly validate!

Reply:
This'll do it: https://github.com/palantir/gradle-baseline/pull/697/files